### ISSUE TITLE
Adding a remote is broken in latest Pharo9

### ISSUE THREAD

**Issue by astares** [pharo-vcs/iceberg#1350]
Click on "Add remote" in recent Pharo 9 image
 - reason is simple: Iceberg code used PluggableTextFieldMorph which was a deprecated method
 - now in recent images the Deprecated80 image was removed.

![image](https://user-images.githubusercontent.com/5980033/75684198-956db200-5c98-11ea-96b1-4efb3e802ca7.png)

**Comment by tesonep** [comment]
@astares Latest Pharo9 does not have references to PluggableTextFieldMorph, that method have been changed, are you loading other library or project?

I have tested on Pharo-9.0.0+build.118.sha.9e01e45c53ba4c135e77336129230bd8bb8b3a80

**Comment by astares** [comment]
@tesonep You are right there are no references. 

I guess it is related to the flow I applied: 

1. I downloaded a new Pharo 9 with Launcher
2. then I used Iceberg repair function to clone again my own fork  "astares/pharo" locally
    (which I used last week before Deprecated80 was thrown out)
3. then I switched to origin (my fork astares) to branch Pharo9.0 
    (which is a little bit elder than regular Pharo project branch) and has the Deprecated80
   Pharo loaded the buildWidget in an old version but not the Deprecated80 

I did this as I wanted to add "pharo-project/pharo" as a remote to update my "astares/pharo" 
locally and then push - but I was not able to do because of the problem above.

I guess the problem is that step 3 "downgrades" the image to my elder/outdated fork 
and then the old #buildWidget is in the image - but the (at that time existing 
DeprecatedPharo80) package is not brought back into the image.

That means if a package is thrown out and you use a newer image but 
"go back in time" with Iceberg as you own fork is not yet updated the thrown
out package do not get back into the image.

I guess this would be too complicated to fix in Iceberg for now - but at least we know where
this is coming from. I use command line git to update my fork and push. This should solve it.
